core/src/main/scala/cats/Eval.scala

```diff
@@ -310,6 +310,30 @@ private[cats] trait EvalInstances extends EvalInstances0 {
       }
     }
 
+  implicit val catsReducibleForEval: Reducible[Eval] =
```

Comment:
none of our instances call `.value` unless the return type requires it. It seems to me we should keep that since otherwise you lose the stack safety if Eval, so that is what I really meant. If you are willing to call `.value`, `Eval` is isomorphic to `Id` I think, but the best practice of `Eval` (though perhaps not required, but the real rule might somewhat complex and difficult to verify) is to only call `.value` once at the end.